state/storagePruningManager/storagePruningManager.go

```diff
@@ -167,6 +169,11 @@ func (spm *storagePruningManager) prune(rootHash []byte, tsm common.StorageManag
 
 	err := spm.removeFromDb(rootHash, tsm)
 	if err != nil {
+		if err == storage.ErrDBIsClosed || strings.Contains(err.Error(), storage.ErrDBIsClosed.Error()) {
```

Comment:
we have a lot of patterns like this. We can have a common, exported function in storage package that returns true if the error is of type ErrDBIsClosed.
Can be left as it is